samples/draw_instanced_ubo.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<!-- Ported from the OpenGL Samples Pack: github.com/g-truc/ogl-samples/blob/master/tests/gl-320-draw-instanced.cpp -->
+<html lang="en">
+
+<head>
+    <title>WebGL 2 Samples - draw_instanced_ubo</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div id="info">WebGL 2 Samples - draw_instanced_ubo</div>
+    <p id="description">This samples demonstrates the use of uniform buffer object</p>
+
+    <script id="vs" type="x-shader/x-vertex">
+        #version 300 es
+        precision highp float;
+        precision highp int;
+
+        layout(std140, column_major) uniform;
+
+        uniform Transform
+        {
+            mat4 MVP[2];
+        } transform;
+
+        layout(location=0) in vec2 pos;
```

Comment:
We should update this to be like the others.

@shrekshao are there any other consistency tweaks needed in this pull request?